chapter1/search-codegen/example_request.py

```
@@ -94,16 +94,28 @@ def make_gpt5_openrouter_request(
+                    "prompt_tokens", usage.get("input_tokens", 0)
+                )
+                output_tokens = usage.get(
+                    "completion_tokens", usage.get("output_tokens", 0)
+                )
+                input_details = usage.get(
+                    "prompt_tokens_details", usage.get("input_tokens_details")
+                )
+                output_details = usage.get(
+                    "completion_tokens_details", usage.get("output_tokens_details")
+                )
                 print(f"\nGPT-5 OpenRouter Usage:")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)